README.rst

```diff
@@ -62,6 +62,82 @@ Quick start
 
 #. Use a double underscore in your translation keys to make use of the filter in the admin (e.g. "header__title" creates a filter called "header"). If you need another separator, set it as DJANGO_TRANSLATOR_CATEGORY_SEPARATOR in your setting file.
 
+
+Custom Models
+-------------
+
+If you find yourself in a situation where you need to use the features of django-translator in a second isolated model, feel free to add one:
+
+#. Create a new model in your app::
+
+    from translator.models import TranslationBase
+
+    class MyCustomTranslation(TranslationBase):
+        pass
+
+
+#. Create a new file `translation.py` and register your model for modeltranslation support::
+
+    from modeltranslation.translator import translator, TranslationOptions
+    from myapp.models import MyCustomTranslation
+
+    class MyCustomTranslationOptions(TranslationOptions):
+        fields = ('description',)
+
+    translator.register(MyCustomTranslation, MyCustomTranslationOptions)
+
+
+#. Add a django admin in `admin.py`::
+
+    from django.contrib import admin
+    from translator.admin import TranslationAdministration
+    from myapp.models import MyCustomTranslation
+
+    @admin.register(MyCustomTranslation)
+    class CustomTranslationAdmin(TranslationAdministration):
+        pass
+
+
+#. Create the two helper functions::
+
+    import six
+    from django.utils.functional import lazy
+    from translator.util import get_translation_for_key
+    from myapp.models import MyCustomTranslation
+
+    def custom_translation(key):
+        return get_translation_for_key(key, model_class=MyCustomTranslation)
+
+    def custom_translation_lazy(item):
+        if len(item) == 0:
+            return ''
+        else:
+            return lazy(get_translation_for_key, six.text_type)(item, MyCustomTranslation)
```

Comment:
Couldn't we add this to the library itself, so a user could use the library function and only supply the model when the function is used? E.g. reuse the `translator_lazy` in `util.py`.

Otherwise, if you were to have several models, you'd end up with several of these helpers (or you'd have to make them generic yourself).